modules/apps/sharing/sharing-web/src/main/resources/META-INF/resources/dynamic_include/bottom.jsp

```diff
@@ -84,8 +53,21 @@ sharingURL.setWindowState(LiferayWindowState.POP_UP);
 				'<%= LanguageUtil.get(resourceBundle, "manage-collaborators") %>'
 			);
 		},
-		[]
-	);
+
+		share: function (classNameId, classPK, title) {
```

Comment:
@wincent I found this spacing between the function keyword and open parenthesis odd. Looking at the codebase I see I have a mix of both. This looks like something we might want to enforce with the liner.. or maybe not. Very low priority. Just saying :)

Reply:
![just-sayin](https://user-images.githubusercontent.com/905006/85409540-3b09a180-b566-11ea-94c2-579718ca53b5.gif)

Reply:
I actually thought about posting this gif but didn't want to violate you exclusive trademark rights

Reply:
😂 Greg gave me permission!

Reply:
In any case, I think this was added in Prettier as [Always add a space after the function keyword](https://prettier.io/blog/2020/03/21/2.0.0.html#always-add-a-space-after-the-function-keyword-3903httpsgithubcomprettierprettierpull3903-by-j-f1httpsgithubcomj-f1-josephfrazierhttpsgithubcomjosephfrazier-sosukesuzukihttpsgithubcomsosukesuzuki-thorn0httpsgithubcomthorn0-7516httpsgithubcomprettierprettierpull7516-by-bakkothttpsgithubcombakkot)

Reply:
Oh no, I was hoping it would be the other way around

![tenor-178268695](https://user-images.githubusercontent.com/156388/85414396-6f6b6680-b542-11ea-97b6-01d255d3f4a3.gif)

Reply:
Yep, it's all Prettier's fault.

> I actually thought about posting this gif but didn't want to violate your exclusive trademark rights

Nothing would make me happier than having *Every. Single. GitHub. Comment.* followed with that gif.